docs/log_collection.md

```diff
@@ -46,22 +46,39 @@ github "DataDog/dd-sdk-ios"
 
 2. Initialize the library with your application context and your [Datadog client token][2]. For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-ios` library as they would be exposed client-side in the iOS application IPA byte code. For more information about setting up a client token, see the [client token documentation][2]:
 
-    {{< tabs >}}
-    {{% tab "US" %}}
-
+{{< site-region region="us" >}}
```

Comment:
Use the same format as [Android](https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_getting_started.md?plain=1#L44L135). Should we provide all region?

Reply:
Bumping this question about region support!

Reply:
We can also add this in another doc card too (thanks for your patience)!